rever/activities/ghrelease.xsh

```diff
@@ -48,6 +49,13 @@ class GHRelease(Activity):
         defaults to ''
     :$GHRELEASE_APPEND: str, string to append to the release notes,
         defaults to ''
+    :$GHRELEASE_ASSETS: list of str or functions, Extra assests to
```

Comment:
Pedantic hat on: shouldn't this be an iterable of str or functions? (since we could pass a tuple)